cpp/src/slice2java/JavaUtil.h

```diff
@@ -114,11 +104,7 @@ namespace Slice
         // package argument matches the entity's package name, then the
         // package is removed from the result.
         //
-        [[nodiscard]] static std::string getUnqualified(
-            const ContainedPtr&,
-            const std::string& = std::string(),
-            const std::string& = std::string(),
-            const std::string& = std::string());
+        [[nodiscard]] static std::string getUnqualified(const ContainedPtr&, const std::string& = std::string());
```

Comment:
Always good to add parameter names to signatures when you update the code.

Reply:
I added parameter names to all the functions in this area of the code, including this one.